x-pack/plugins/security_solution/public/common/components/hover_actions/index.tsx

```diff
@@ -217,10 +218,12 @@ export const HoverActions: React.FC<Props> = React.memo(
     const isCaseView = scopeId === TimelineId.casePage;
     const isTimelineView = scopeId === TimelineId.active;
     const isAlertDetailsView = scopeId === TimelineId.detectionsAlertDetailsPage;
+    // TODO Provide a list of disabled/enabled actions as props
```

Comment:
With the upcoming tech debt changes we will be able to define the list of the available actions for the specific consumer like entityAnaliticsPage